docs/2.8.0/docs/kubernetes/03-deploy-dependencies/03-deploy-traefik-ingress-controller.rst

```diff
@@ -23,17 +23,17 @@ Installation guide
 
       helm repo add traefik https://traefik.github.io/charts
 
-#. Confirm Helm chart availability
+#. Confirm Helm chart availability:
 
-After the previous step we should be able to search the repo for the Traefik chart:
+After the previous step, you should be able to search the repo for the Traefik chart:
 
 .. code-block:: bash
 
      helm search repo traefik
 
 Expected output:
 
-.. code-block:: console
+.. code-block:: bash
```

Comment:
It seems defensible to use `bash` for input and `console` for output, but I'm not sure about our standards here.

Reply:
(see comment above about possible CSS issues with `console`)

Reply:
going to leave the style as is for now. If we get `console` working in the future, we can change it back then.

Reply:
The problem is it is harder to fix if we don't have the error online, plus now we'll have to go back and fix these `code-block` styles again.

Reply:
Agree, though feeling that people seeing a black box with no text might be even worse.